### ISSUE TITLE
Cannot use same port "port is already allocated"

### ISSUE THREAD

**Issue by gurre** [thugs-rumal/rumal_docker#1]
Both the frontend and backend uses port 8000. This leads to an error trying to startup.

```
$ git clone https://github.com/thugs-rumal/rumal_docker/
...
$ docker-compose up
Creating rumaldocker_backend_1
Creating rumaldocker_frontend_1

ERROR: for frontend  driver failed programming external connectivity on endpoint rumaldocker_frontend_1 (62d3a7a6e4e98f3dcd0eba49557620e4237dbf76b0504955dd0137817b7c910b): Bind for 0.0.0.0:8000 failed: port is already allocated
Traceback (most recent call last):
  File "<string>", line 3, in <module>
  File "compose/cli/main.py", line 63, in main
AttributeError: 'ProjectError' object has no attribute 'msg'
docker-compose returned -1
```

```
$ uname -a
Linux ubuntu 3.13.0-86-generic #131-Ubuntu SMP Thu May 12 23:33:13 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
$ docker version
Client:
 Version:      1.11.1
 API version:  1.23
 Go version:   go1.5.4
 Git commit:   5604cbe
 Built:        Tue Apr 26 23:30:23 2016
 OS/Arch:      linux/amd64

Server:
 Version:      1.11.1
 API version:  1.23
 Go version:   go1.5.4
 Git commit:   5604cbe
 Built:        Tue Apr 26 23:30:23 2016
 OS/Arch:      linux/amd64
```

**Comment by Dennisparchkov** [comment]
Hi @gurre, iv been working on rumal_docker and these issues (#2, #3 , #4) should be fixed in the PR #5

**Comment by gurre** [comment]
Awesome. Good job so far!👍🏻
